book/3-classification.tex

```diff
@@ -268,4 +268,74 @@ \subsection{Phi Coefficient}
 \clearpage
 \thispagestyle{classificationstyle}
 \section{MCC}
-\subsection{Matthew's Correlation Coefficient}
\ No newline at end of file
+\subsection{Matthew's Correlation Coefficient}
+
+
+% ---------- EC ----------
+\clearpage
+\thispagestyle{classificationstyle}
+\section{EC}
+\subsection{Expected Cost}
+
+The Expected Cost (EC) metric is designed to evaluate the performance of decision-making systems. It is the expected value of a cost function that assigns a penalty to each combination of the system's decision and the true class of a sample.
+
+\begin{center}
+\tikz{
+\node[inner sep=2pt, font=\Large] (a) {
+{
+$\displaystyle
+EC = \frac{1}{{\color{red}N}} \sum_{t=1}^{N} {\color{purple}C}({\color{cyan}h_t}, {\color{orange}d_t})
+$
+}
+};
+\draw[-latex,red, semithick] ($(a.south)+(-0.7,0.2)$) to[bend left=30] node[pos=1, left] {number of samples} +(-0.8,-0.5); % Increased length
+\draw[-latex,cyan, semithick] ($(a.north)+(1.5,-0.1)$) to[bend left=30] node[pos=1, right] {True class} +(0.8,0.5); % Increased length
```

Comment:
reduce the space between the variable and the arrow.

<img width="559" alt="Screenshot 2024-10-15 at 11 42 00" src="https://github.com/user-attachments/assets/9b0e878e-ac7a-4ccd-a4ae-11f8fcb07dde">

Similar to before use the formula colors defined in main.tex

Reply:
We also need and arrow explaining what is C